app/jobs/delius_import_job.rb

```diff
@@ -149,4 +163,14 @@ def for_each_record(processor)
       yield(record)
     end
   end
+
+  def active_record?(ldu_code)
+    return false if ldu_code.match('N\d{2}OMIC').present?
+
+    true
+  end
+
+  def shadow_record?(ldu_code)
+    ldu_code.match('N\d{2}OMIC').present?
```

Comment:
I think we could improve the readability of this line by using the [`.match?` method](https://ruby-doc.org/core-2.6.3/Regexp.html#method-i-match-3F) in place of `.match().presence?`:

```suggestion
    ldu_code.match?('N\d{2}OMIC')
```

The [`match` method](https://ruby-doc.org/core-2.6.3/Regexp.html#method-i-match) returns a `MatchData` object which gives us the results of the regex match. This is useful, for example, when we want to extract something from the original string. But in our case, we're not interested in extracting anything from the original string – we just want to know whether it matches or not.

So for that, we can use the `match?` method. This will simply give us a boolean `true` or `false` response, rather than a `MatchData` object that we then need to inspect to see if it has 'presence' or if it's empty.

Both approaches ultimately achieve the same thing, so this is mostly just a matter of preference. No need to change if you're happy as it is.